lib/Scraper/IScraper.php

```diff
@@ -26,10 +26,10 @@ public function scrape(string $url): bool;
     /**
      * Get the scraped content
      *
-     * @return string
+     * @return string, null if failed
```

Comment:
```suggestion
     * @return string|null
```

Reply:
Its just better for IDEs like so, I guess we could also just do @return ?string, but not sure about the implementations in IDEs.